src/options/views/script-item.vue

```diff
@@ -4,86 +4,88 @@
     :class="{ disabled: !script.config.enabled, removed: script.config.removed }"
     :draggable="draggable"
     @dragstart.prevent="onDragStart">
-    <img class="script-icon hidden-xs" :src="safeIcon">
-    <div class="script-info flex">
-      <div class="script-name ellipsis flex-auto" v-text="script.$cache.name"></div>
-      <tooltip
-        v-if="author"
-        :content="i18n('labelAuthor') + script.meta.author"
-        class="script-author ml-1 hidden-sm"
-        align="end">
-        <icon name="author"></icon>
-        <a
-          v-if="author.email"
-          class="ellipsis ml-1"
-          :href="`mailto:${author.email}`"
-          v-text="author.name"
-        />
-        <span class="ellipsis ml-1" v-else v-text="author.name"></span>
-      </tooltip>
-      <tooltip class="ml-1 hidden-sm" :content="updatedAt.title" align="end">
-        <span v-text="script.meta.version ? `v${script.meta.version}` : ''"></span>
-        <span class="ml-1" v-text="updatedAt.show"></span>
-      </tooltip>
-      <div v-if="script.config.removed" class="ml-1">
-        <tooltip :content="i18n('buttonRestore')" placement="left">
-          <span class="btn-ghost" @click="onRestore">
-            <icon name="undo"></icon>
-          </span>
-        </tooltip>
-      </div>
-    </div>
-    <div class="script-buttons flex">
-      <div class="flex-auto flex flex-wrap">
-        <tooltip :content="i18n('buttonEdit')" align="start">
-          <span class="btn-ghost" @click="onEdit">
-            <icon name="code"></icon>
-          </span>
-        </tooltip>
-        <tooltip :content="labelEnable" align="start">
-          <span class="btn-ghost" @click="onEnable">
-            <icon :name="`toggle-${script.config.enabled ? 'on' : 'off'}`"></icon>
-          </span>
-        </tooltip>
+    <template v-if="canRender">
```

Comment:
This huge block of changes is just an indent. You may want to "hide whitespace changes" in the config menu at the top of the page if you're viewing this on github.